### PR TITLE
DEV: Improve logging and fix some formatting errors

### PR DESCRIPTION
**Pull request by tgxworld**

### CODE REVIEW

File: plugin.rb [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # frozen_string_literal: true
 
 # name: discourse-oauth2-basic
-# about: Allows users to login to your forum using a basic OAuth2 provider.
```
Comment (tgxworld) [inline plugin.rb:4]: syntax tree error

File: plugin.rb [hunk 2]
```diff
@@ -50,7 +50,7 @@ def recurse(obj, keys)
 class OAuth2FaradayFormatter < Faraday::Logging::Formatter
   def request(env)
     warn <<~LOG
-      OAuth2 Debugging: request #{env.method.upcase} #{env.url.to_s}
```
Comment (tgxworld) [inline plugin.rb:53]: rubocop error

File: plugin.rb [hunk 3]
```diff
@@ -64,7 +64,7 @@ def response(env)
     warn <<~LOG
       OAuth2 Debugging: response status #{env.status}
 
-      From #{env.method.upcase} #{env.url.to_s}
```
Comment (tgxworld) [inline plugin.rb:67]: rubocop error

File: plugin.rb [hunk 4]
```diff
@@ -236,8 +236,12 @@ def fetch_user_details(token, id)
...
       user_json request: #{user_json_method} #{user_json_url}
 
-      response:
-      #{user_json_response.to_yaml}
+      request headers: #{headers}
+
+      response status: #{user_json_response.status}
+
+      response body:
+      #{user_json_response.body}
```
Comment (tgxworld) [inline plugin.rb:244]: Things added to the logs

**Review by SamSaffron — APPROVED** [review]

**Pull request merged**